src/main/java/org/mule/extension/oauth2/api/authorizationcode/DefaultAuthorizationCodeGrantType.java

```diff
@@ -159,6 +159,14 @@ public class DefaultAuthorizationCodeGrantType extends AbstractGrantType {
   @Optional(defaultValue = "true")
   private boolean encodeClientCredentialsInBody;
 
+  @Inject
```

Comment:
Should this be injected in an API class? (maybe yes in this case, just want to double check)

Reply:
I've just moved code. See https://github.com/mulesoft/mule-oauth-module/blob/main/src/main/java/org/mule/extension/oauth2/api/authorizationcode/package-info.java or any package-info.java classes in api packages.